panther_lights/CMakeLists.txt

```diff
@@ -221,6 +225,12 @@ if(BUILD_TESTING)
   target_link_libraries(${PROJECT_NAME}_test_lights_controller_node
                         lights_controller_node_component yaml-cpp)
 
+  # Integration tests
+  option(TEST_INTEGRATION "Run integration tests" OFF)
+  if(TEST_INTEGRATION)
+    add_ros_test(test/integration/panther_lights.test.py)
+  endif()
+
```

Comment:
should we move this outside BUILD_TESTING? This way we could run only integration tests on remote.
Unit test can be done on local machine and building them on built-in computer can take a lot of time.

Reply:
As we discussed, it is the most appropriate behavior for now.